Assets/ScriptableObjects/Dialogue/DialogueController.cs

```diff
@@ -160,17 +164,28 @@ private IEnumerator AnimateDialogueNode(DialogueNode dialogueNode, DialogueOptio
             _currentDialogueCoroutine = null;
 
             if (i == dialogueNode.sentences.Length - 1)
-                SetHintText("Close [E]");
+            {
+                if (dialogueNode.HasChoices())
+                {
+                    _currentChoiceCoroutine = WaitForChoice(dialogueNode, choiceCallback);
+                    yield return StartCoroutine(_currentChoiceCoroutine);
+                    SetHintText("Select [E]");
+                }
+                else
+                    SetHintText("Close [E]");
```

Comment:
Not sure if we want the option to change controls, if so these should be dynamic.

Reply:
will do later if we have time at the end, created a ticket for it #95